<ui version="4.0" >
 <class>OpenFileFromWebDlg</class>
 <widget class="QDialog" name="OpenFileFromWebDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Open file from web...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QToolButton" name="back_button" >
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="st4.qrc" >
         <normaloff>:/images/previous.png</normaloff>:/images/previous.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="forward_button" >
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="st4.qrc" >
         <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="url" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWebView" name="webView" >
     <property name="url" >
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="status_message" >
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="st4.qrc" />
 </resources>
 <connections/>
</ui>
